docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -42,14 +43,36 @@ ifeval::["{build}" == "downstream"]
 STORAGE_CLASS=local-storage
 MARIADB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-mariadb-rhel9:18.0
 endif::[]
-# Replace with your environment's MariaDB Galera cluster VIP and backend IPs:
-SOURCE_MARIADB_IP=172.17.0.2
-declare -A SOURCE_GALERA_MEMBERS
-SOURCE_GALERA_MEMBERS=(
+declare -A TRIPLEO_PASSWORDS
+TRIPLEO_PASSWORDS[default]='~/overcloud-deploy/overcloud/overcloud-passwords.yaml'
+TRIPLEO_PASSWORDS[cell1]='~/cell1/cell1-passwords.yaml'
+TRIPLEO_PASSWORDS[cell2]='~/cell2/cell2-passwords.yaml'
+# ...
+
+# Replace with your environment's main overcloud stack MariaDB Galera cluster VIP:
+declare -A SOURCE_MARIADB_IP
+SOURCE_MARIADB_IP[default]=172.17.0.90
+SOURCE_MARIADB_IP[cell1]=172.17.0.91
+SOURCE_MARIADB_IP[cell2]=172.17.0.92
+# ...
+
+# Replace with all members data for each MariaDB Galera cluster in the overcloud and additional cell stacks
+declare -A SOURCE_GALERA_MEMBERS_DEFAULT
```

Comment:
this is a bit clumsy, but we don't really want to deal with nested dictionaries in bash, do we?